### ISSUE TITLE
Problem: upgrade to prost 0.8.0 fails

### ISSUE THREAD

**Issue by tomtau** [crypto-com/tmkms-light#124]
```
error[E0599]: no function or associated item named `decode_length_delimited` found for struct `tendermint_proto::privval::Message` in the current scope
523
  --> src/rpc.rs:41:32
524
   |
525
41 |         let msg = PrivMessage::decode_length_delimited(msg.as_ref())
526
   |                                ^^^^^^^^^^^^^^^^^^^^^^^ function or associated item not found in `tendermint_proto::privval::Message`
527
   |
528
   = help: items from traits can only be used if the trait is in scope
529
   = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
530
           `use prost::message::Message;`
531

532
error[E0599]: no method named `encode_length_delimited` found for struct `tendermint_proto::privval::Message` in the current scope
533
   --> src/rpc.rs:191:14
534
    |
535
191 |             .encode_length_delimited(&mut buf)
536
    |              ^^^^^^^^^^^^^^^^^^^^^^^ method not found in `tendermint_proto::privval::Message`
537
    | 
538
   ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/prost-0.7.0/src/message.rs:65:8
539
    |
540
65  |     fn encode_length_delimited<B>(&self, buf: &mut B) -> Result<(), EncodeError>
541
    |        ----------------------- the method is available for `tendermint_proto::privval::Message` here
542
    |
543
    = help: items from traits can only be used if the trait is in scope
544
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
545
            `use prost::message::Message;`
546

547
error: unused import: `prost::Message`
548
 --> src/rpc.rs:7:5
549
  |
550
7 | use prost::Message as _;
551
  |     ^^^^^^^^^^^^^^
552
  |
553
  = note: `-D unused-imports` implied by `-D warnings`
554

555
error: aborting due to 3 previous errors
```

**Comment by linfeng-crypto** [comment]
the crate `tendermint-proto` use prost 0.7.0, and doesn't expose the prost, so the version of prost should match to 0.7.0.

**Comment by tomtau** [comment]
https://github.com/informalsystems/tendermint-rs/blob/master/CHANGELOG.md#v0210

> A security issue was reported in prost v0.7, and we attempted to upgrade to v0.8, but we are still awaiting one bug fix in v0.8 before we can upgrade. The moment that is fixed in prost, we will upgrade to v0.8 and provide another tendermint-rs release.
